astronomer/providers/snowflake/operators/snowflake_sql_api.py

```diff
@@ -0,0 +1,153 @@
+import typing
+from datetime import timedelta
+from typing import Any, Dict, List, Optional, Union
+
+from airflow.exceptions import AirflowException
+from airflow.providers.snowflake.operators.snowflake import SnowflakeOperator
+from airflow.utils.context import Context
+
+from astronomer.providers.snowflake.hooks.snowflake_sql_api import (
+    SnowflakeSqlApiHookAsync,
+)
+from astronomer.providers.snowflake.triggers.snowflake_sql_api_trigger import (
+    SnowflakeSqlApiTrigger,
+)
+
+
+class SnowflakeSqlApiOperatorAsync(SnowflakeOperator):
```

Comment:
just a thought, why didn't we keep it in snowflake.py?

Reply:
Moved it in snowflake.py file